frontend/src/app/shared/components/sidenav/sidenav.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../../../sass/color';
+
+sidenav-layout {
+    box-sizing: border-box;
+    display: block;
+    height: 100%;
+}
+sidenav-layout.sidenav-shown {
```

Comment:
No that won't work because the `sidenav-shown` has to be on the sidenav-layout. If I put it in the sidenav-layout block it expects the `sidenav-shown` as a child but i don't want that.